snooty/gizaparser/test_extracts.py

```diff
@@ -140,3 +145,77 @@ def test_external_cycle() -> None:
             FileId("includes/extracts-test2.yaml"): [FailedToInheritRef],
             FileId("includes/extracts-test1.yaml"): [FailedToInheritRef],
         }
+
+
+def test_partial_unmarshaling_error() -> None:
+    with make_test(
+        {
+            Path(
+                "source/includes/extracts-test1.yaml"
+            ): """
+ref: bypassDocumentValidation-db.collection.aggregate
+inherit:
+  ref: _bypassDocValidation
+  file: extracts-bypassDocumentValidation-base.yaml
+replacement:
+  role: ":method:`db.collection.aggregate()`"
+  interface: "method"
+post: |
+  Document validation only occurs if you are using the
+  :pipeline:`$out` operator in your aggregation operation.
+---
+ref: bypassDocumentValidation-aggregate
+content: "Okay"
+...
+"""
+        }
+    ) as result:
+        assert list(result.pages.keys()) == [
+            FileId("includes/extracts/bypassDocumentValidation-aggregate.rst")
+        ]
+        assert {k: [type(d) for d in v] for k, v in result.diagnostics.items()} == {
+            FileId("includes/extracts-test1.yaml"): [UnmarshallingError]
```

Comment:
Perhaps unimportant to the fix in this PR but curious for consistency's sake, should the error be `unmarshaling` or `unmarshalling`?

Reply:
two L's: unmarshalling. Commit message is a spelling mistake